cake/builder.cake

```diff
@@ -0,0 +1,22 @@
+//////////////////////////////////////////////////////////////////////
+// EXECUTION
+//////////////////////////////////////////////////////////////////////
+
+public Builder Build => CommandLineOptions.Usage
+    ? new Builder(() => Information(HelpMessages.Usage))
+    : new Builder(() => RunTarget(CommandLineOptions.Target.Value));
+
+public class Builder
```

Comment:
I'm surprised there's not built-in functionality in cake to support do something like this.